lib/src/models/profile.dart

```diff
@@ -31,7 +31,15 @@ class Profile {
   final Account account;
   final ProfileType type;
 
+  /// Duration Allowed for this Account in minutes
+  final int durationAllowed;
+  /// Duration Used for this Account in minutes
+  final int durationUsed;
+
   Profile.fromJson(Map<String, dynamic> json)
       : account = Account.fromJson(json['account']),
-        type = ProfileType.fromName(json['userType']);
+        type = ProfileType.fromName(json['userType']),
+        durationAllowed = _convertSecondsToMinutes(json['durationAllowed']),
+        durationUsed = _convertSecondsToMinutes(json['durationUsed']);
```

Comment:
This doesn't feel like the right place to expose this data. A couple other concerns:
1. This means you'll have to refresh the profile after every service request, since the `durationUsed` could have changed.
2. Converting to minutes means you're losing information. Platform will allow you to make a call if you have at least 10 seconds left, but that'll show up as 0 minutes.

Reply:
I converted to minutes to keep consistency with the other API which return the time for the user profile (in minutes). As for the place... I agree. This profile information is a little "all over the place" to my eyes and I didn't think this was the time to mingle with this.

Thanks for your comment though as I realize we have an issue where the Usage should read the profile information from the API before displaying to ensure fresh data.